lib/reporters/tap.js

```diff
@@ -7,7 +7,7 @@ function getSourceFromStack(stack, index) {
 		.split('\n')
 		.slice(index, index + 1)
 		.join('')
-		.replace(/^\s+at /, '');
```

Comment:
I think it's because the `test` function [already has](https://github.com/sindresorhus/ava/blob/master/lib/reporters/tap.js#L61) `'at'` in it. Either way, if you add `at` to the Regex the space trimming does't work.